lib/tasks/generate_fake_users.rake

```diff
@@ -0,0 +1,23 @@
+require 'faker'
+
+desc 'Generate 10 fake user account if less than 10 users'
+task generate_fake_users: :environment do
+  if User.count < 10
+    sports = %w(Fussball Tennis Volleyball Golf Tischtennis)
+
+    10.times do
+      sports.shuffle
+      password = Faker::Internet.password
```

Comment:
Do we want to be able to log in with the credentials of these users? If so, can we access the password somehow?

Reply:
Nop, this was not a requirement.

Reply:
However, Jonas as PO mentioned some potential changes. lets see